src/FITSIO.jl

```diff
@@ -147,6 +147,23 @@ include("header.jl")  # FITSHeader methods
 include("image.jl")  # ImageHDU methods
 include("table.jl")  # TableHDU & ASCIITableHDU methods
 
+"""
+    libcfitsio_version() -> VersionNumber
+
+### Purpose ###
+
+Return the version of the underlying CFITSIO library
+
+### Example ###
+
+```julia
+julia> using FITSIO
```

Comment:
I think `using FITSIO` can be omitted from examples. Otherwise it would have to be the first line of every single example. Is it necessary for doctests to run correctly?